icons/couchbase/couchbase-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
2l-0.8674 1.8683c-0.42236-0.26689-1.0451-0.55603-2.0017-0.55603-1.5124 0-2.3129 1.1788-2.3129 2.9358 0 1.7568 0.75598 3.0248 2.3798 3.0248 1.1121 0 1.7568-0.40034 2.046-0.57827l0.82292 1.8236c-0.44482 0.28936-1.3345 0.84539-3.1138 0.84539zm10.409-0.22241v-6.2053c0-0.95637-0.33362-1.4234-1.2455-1.4234-0.97861 0-1.7348 0.91189-1.935 1.1565v6.4499h-2.7134v-14.234h2.7134v5.7382c0.4893-0.53379 1.4234-1.379 3.0026-1.379 2.1354 0 2.8913 1.3567 2.8913 3.136v6.7391h-2.7134zm7.651-2.1129c0.17793 0.06672 0.57827 0.19995 1.29 0.19995 1.4012 0 2.2686-1.0231 2.2686-3.0026 0-1.7793-0.60051-2.8469-1.9795-2.8469-0.80068 0-1.4232 0.51155-1.5791 0.71172zm0-12.121v5.3379c0.44482-0.44482 1.29-1.0009 2.3576-1.0009 2.4023 0 4.0924 1.6458 4.0924 4.982 0 3.3139-2.2241 5.1379-5.1599 5.1379-2.2016 0-3.5361-0.55603-4.0256-0.77844v-13.701h2.7357zm12.922 9.9863h-0.33361c-1.4899 0-2.5353 0.44482-2.5353 1.5346 0 0.68948 0.55603 0.97839 1.1565 0.97839 0.91188 0 1.4457-0.53356 1.7123-0.80068zm0.28914 4.2481c-0.089-0.24488-0.13367-0.73418-0.15569-1.0233-0.42258 0.51155-1.3345 1.2677-2.6914 1.2677-1.6456 2.23e-4 -2.9134-0.97839-2.9134-2.7132 0-2.5132 2.58-3.4249 5.1377-3.4249h0.33361v-0.53401c0-0.80068-0.33361-1.29-1.5344-1.29-1.2457 0-2.0907 0.64499-2.4465 0.88965l-1.1788-1.6458c0.57827-0.53379 1.868-1.4012 3.9365-1.4012 2.6689 0 3.9367 1.0231 3.9367 3.6031v3.3807c0 1.3345 0.13367 2.2684 0.26689 2.8913zm7.3173 0.22241c-1.7348 0-2.8469-0.51177-3.4474-0.91211l0.91211-1.8905c0.37788 0.26712 1.312 0.8009 2.3796 0.8009 1.0008 0 1.5569-0.28936 1.5569-0.82292 0-0.62275-1.0896-0.8009-2.4688-1.4681-1.3345-0.64477-2.1129-1.3787-2.1129-2.8469 0-1.8238 1.4457-2.9803 3.6476-2.9803 1.6456 0 2.6689 0.51155 3.1138 0.7562l-0.93412 1.846c-0.3781-0.22241-1.1345-0.60051-2.0462-0.60051-0.91166 0-1.3122 0.31138-1.3122 0.82292 0 0.62275 0.88965 0.77844 2.0017 1.2233 1.5126 0.62297 2.6022 1.3345 2.6022 2.9358-0.0445 2.0686-1.5124 3.1362-3.8922 3.1362zm11.187-6.2275c-0.0222-1.0676-0.44482-1.935-1.6011-1.935-1.0678 0-1.6905 0.68948-1.8682 1.935zm-1.0231 4.2258c1.3122 0 2.0239-0.42258 2.5132-0.66723l0.8674 1.7348c-0.64499 0.35586-1.6236 0.93413-3.6253 0.93413-3.2474 0-5.0932-2.0242-5.0932-5.1155 0-3.0693 2.1129-5.0043 4.8041-5.0043 3.047 0 4.5152 2.1129 4.1816 5.716h-6.072c0.1112 1.4457 0.8674 2.402 2.4243 2.402z"/><path d="m14.724 49.276c-8.118 0-14.724 6.5834-14.724 14.724 0 8.118 6.5834 14.724 14.724 14.724 8.118 0 14.724-6.5834 14.724-14.724 0-8.1403-6.6056-14.724-14.724-14.724zm9.9418 17.304c0 0.88964-0.51155 1.6681-1.5124 1.846-1.7348 0.31115-5.3824 0.4893-8.4294 0.4893-3.047 0-6.6946-0.17815-8.4294-0.4893-1.0009-0.17793-1.5124-0.95637-1.5124-1.846v-5.7382c0-0.88965 0.68948-1.7126 1.5124-1.846 0.51155-0.08896 1.7126-0.17793 2.6467-0.17793 0.35586 0 0.64499 0.26689 0.64499 0.68948v4.0256l5.1599-0.11121 5.1599 0.11121v-4.0256c0-0.42258 0.28913-0.68948 0.64499-0.68948 0.93413 0 2.1351 0.08896 2.6467 0.17793 0.84516 0.13345 1.5124 0.95637 1.5124 1.846-0.04448 1.8905-0.04448 3.8255-0.04448 5.7382z"/></svg>
```

Comment:
Sinc ethis is a plain icon it needs to have a unified path (a.k.a only a single `<path>` element)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M14.725 49.275C6.607 49.275 0 55.86 0 64c0 8.118 6.584 14.725 14.725 14.725 8.118 0 14.722-6.584 14.722-14.725 0-8.14-6.604-14.725-14.722-14.725zm65.343 7.053v14.233h2.713v-6.45c.2-.244.957-1.156 1.936-1.156.912 0 1.244.468 1.244 1.424v6.183h2.715v-6.74c0-1.779-.757-3.136-2.893-3.136-1.579 0-2.512.847-3.002 1.38v-5.738h-2.713zm10.809 0v13.7c.49.222 1.824.779 4.025.779 2.936 0 5.16-1.825 5.16-5.139 0-3.336-1.689-4.98-4.091-4.98-1.068 0-1.913.555-2.358 1v-5.36h-2.736zm-46.639.822c-4.47 0-6.828 3.224-6.828 6.805 0 3.603 2.268 6.852 6.783 6.852 1.935 0 3.515-.779 4.182-1.38l-1.18-2.025c-.69.512-1.533 1.002-2.957 1.002-2.513 0-3.76-2.046-3.76-4.515 0-2.402 1.291-4.336 3.715-4.336 1.201 0 2 .289 2.735.756l1.29-2.114c-1.045-.667-2.267-1.045-3.98-1.045zM8.941 58.818c.356 0 .645.267.645.69v4.025l5.16-.111 5.16.111v-4.025c0-.423.289-.69.645-.69.934 0 2.135.09 2.646.178.845.134 1.514.956 1.514 1.846-.045 1.89-.045 3.825-.045 5.738 0 .89-.51 1.668-1.512 1.846-1.735.31-5.382.49-8.43.49-3.046 0-6.694-.18-8.43-.49-1-.178-1.513-.956-1.513-1.846v-5.738c0-.89.691-1.713 1.514-1.846.511-.089 1.712-.178 2.646-.178zm44.883 1.868c-3.38 0-4.85 2.602-4.85 5.093s1.47 5.028 4.85 5.028c3.403 0 4.892-2.649 4.914-5.118 0-2.49-1.533-5.003-4.914-5.003zm21.819 0c-2.959 0-5.118 1.825-5.118 5.072 0 3.024 1.914 5.049 5.028 5.049 1.779 0 2.668-.557 3.113-.846l-.822-1.824c-.29.178-.935.578-2.047.578-1.624 0-2.379-1.267-2.379-3.024 0-1.757.8-2.937 2.312-2.937.957 0 1.58.29 2.002.557l.868-1.868c-.467-.289-1.4-.757-2.957-.757zm38.964.002c-2.201 0-3.646 1.154-3.646 2.978 0 1.468.779 2.203 2.113 2.848 1.38.667 2.469.846 2.469 1.468 0 .534-.556.823-1.557.823-1.067 0-2.003-.534-2.38-.801l-.913 1.89c.6.4 1.713.913 3.448.913 2.38 0 3.848-1.068 3.892-3.137 0-1.601-1.089-2.313-2.601-2.936-1.112-.444-2.002-.6-2.002-1.222 0-.512.4-.825 1.312-.825s1.667.38 2.045.602l.936-1.846c-.445-.244-1.47-.755-3.116-.755zm9.164 0c-2.69 0-4.804 1.934-4.804 5.003 0 3.092 1.846 5.116 5.094 5.116 2.001 0 2.98-.578 3.625-.934l-.868-1.736c-.489.244-1.201.668-2.513.668-1.557 0-2.313-.957-2.424-2.403h6.072c.334-3.603-1.135-5.715-4.182-5.715zm-18.46.021c-2.069 0-3.358.867-3.936 1.4l1.18 1.647c.355-.245 1.2-.889 2.445-.889 1.2 0 1.535.489 1.535 1.29v.532h-.334c-2.558 0-5.138.913-5.138 3.426 0 1.735 1.268 2.713 2.914 2.713 1.356 0 2.268-.756 2.691-1.267.022.289.067.778.156 1.023h2.692c-.134-.623-.268-1.556-.268-2.89v-3.382c0-2.58-1.269-3.603-3.937-3.603zm-45.082.2v6.74c0 1.779.757 3.134 2.892 3.134a3.838 3.838 0 0 0 3.113-1.512c.022.4.11 1.023.2 1.29h2.691c-.133-.578-.266-1.557-.266-2.825v-6.804h-2.714v6.449c-.2.222-.934 1.156-1.913 1.156-.912 0-1.246-.467-1.246-1.424v-6.205H60.23zm63.498 1.736c1.156 0 1.579.866 1.601 1.933h-3.469c.178-1.245.8-1.933 1.868-1.933zm-69.948.066c1.468 0 2.046 1.289 2.069 3.068 0 1.78-.49 2.98-1.957 2.98-1.446 0-2.026-1.267-2.026-3.046 0-1.78.469-3.002 1.914-3.002zm41.412.11c1.38 0 1.98 1.068 1.98 2.847 0 1.98-.867 3.002-2.269 3.002-.711 0-1.11-.133-1.289-.2v-4.937c.156-.2.778-.713 1.578-.713zm11.01 3.515h.334v1.713c-.266.267-.8.8-1.713.8-.6 0-1.156-.288-1.156-.978 0-1.09 1.045-1.535 2.535-1.535z"/></svg>
```